packages/minter-contracts/test/bootstrap-sandbox.ts

```diff
@@ -17,22 +18,16 @@ async function flextesaKeys(): Promise<TestKeys> {
   const alice = await InMemorySigner.fromSecretKey(
     'edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq',
   );
-  return { bob, alice };
-}
-
-async function testnetKeys(): Promise<TestKeys> {
-  const bob = await InMemorySigner.fromSecretKey(
-    'edskRfLsHb49bP4dTpYzAZ7qHCX4ByK2g6Cwq2LWqRYAQSeRpziaZGBW72vrJnp1ahLGKd9rXUf7RHzm8EmyPgseUi3VS9putT',
-  );
-  const alice = await InMemorySigner.fromSecretKey(
-    'edskRqb8GgnD4d2B7nR3ofJajDU7kwooUzXz7yMwRdLDP9j7Z1DvhaeBcs8WkJ4ELXXJgVkq5tGwrFibojDjYVaG7n4Tq1qDxZ',
+  const eve = await InMemorySigner.fromSecretKey(
+    'edsk2rKA8YEExg9Zo2qNPiQnnYheF1DhqjLVmfKdxiFfu5GyGRZRnb',
```

Comment:
Is this a new bootstrapped account in flextesa? If so, should we also update flextesa image in `start-sandbox.sh`?

Reply:
Fixed